src/bundle/Resources/public/js/CKEditor/embed/image/embed-image-editing.js

```diff
@@ -62,7 +62,7 @@ class IbexaEmbedImageEditing extends Plugin {
         const children = viewElement.getChildren();
 
         for (const child of children) {
-            if (child.getAttribute('data-ezelement') === settingName) {
+            if (child.getAttribute?.('data-ezelement') === settingName) {
```

Comment:
Why it doesn't have the getAttribute method? What is the child?

Reply:
Our image element has two children.
The first one is Element and it has the getAttribute function, but the second one is Text (https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_view_text-Text.html) and don't have this function

Reply:
Hmm the interesting part is why it works on adding element/upcasting on editing but not working on drag and drop.
Also it would be better to check if child is an  element and not if has the method: `child.is('element')`